File: app/src/main/java/com/said/homework/base/data/database/AppDatabaseOpenHelper.java
package com.said.homework.base.data.database;

import android.content.Context;

import org.greenrobot.greendao.database.Database;

import database.DaoMaster;

/**
 * Created by Ahmed Sa'eed on 27,November,2020
 */
public class AppDatabaseOpenHelper extends DaoMaster.OpenHelper {

    public AppDatabaseOpenHelper(Context context, String name) {
        super(context, name);
    }

    @Override
    public void onCreate(Database db) {
        super.onCreate(db);
    }
}

File: app/src/main/java/com/said/homework/base/data/database/AppDatabaseOpenHelper.kt
package com.said.homework.base.data.database

import android.content.Context
import database.DaoMaster.OpenHelper
import org.greenrobot.greendao.database.Database

/**
 * Created by Ahmed Sa'eed on 27,November,2020
 */
class AppDatabaseOpenHelper(context: Context?, name: String?) : OpenHelper(context, name) {
    override fun onCreate(db: Database) {
        super.onCreate(db)
    }
}